### PR TITLE
New build with updated dependencies

### PR DESCRIPTION
**Pull request by kprigel**
Updated dependencies to resolve hls.js audio issue introduced in hls.js 5

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -7,10 +7,10 @@
     "example": "examples"
   },
   "dependencies": {
-    "hls.js": "^0.5.44",
-    "video.js": "^5.13.0",
-    "videojs-contrib-hls": "^3.5.3",
-    "videojs-contrib-media-sources": "^3.1.5"
+    "hls.js": "^0.6.7",
+    "video.js": "^5.11.9",
```
Comment (shacharz) [inline package.json]: why downgrade video.js?

Comment (kprigel) [inline package.json]: I was using the latest official release according to npm:
https://www.npmjs.com/package/video.js

And github:
https://github.com/videojs/video.js/releases

Comment (boushley) [inline package.json]: The `5.13.0` release of video.js has been deprecated.

You can see it marked as "Ignored" in their Changelog: https://github.com/videojs/video.js/blob/master/CHANGELOG.md#5130-2016-08-25

In addition when installing video.js 5.13.0 as a dependency of this project there is a deprecation warning `npm WARN deprecated video.js@5.13.0: Bad release. Please use 5.12`

Any note on when we can expect this to be merged?

**Comment by boushley** [comment]
@kprigel it also appears that the `dist/` folder is not normally checked into the repo. It should probably be added to the `.gitignore` if we want to continue like that, but adding the `dist` file seems out of scope for the change you're pushing for here.

**Comment by boushley** [comment]
Also, so that GitHub can do its magic, this would resolve #7

**Pull request merged**